2024/week1_cryptographic_basics.md

```diff
@@ -522,6 +522,7 @@ If you would like to know more about it, you can check out the following resourc
 
 - **[Exploring Elliptic Curve Pairings by Vitalik Buterin (optional)](https://medium.com/@VitalikButerin/exploring-elliptic-curve-pairings-c73c1864e627)** - This resource builds upon the knowledge you learned above regarding elliptic curves and sets the stage for the topics discussed below. It is an excellent introduction to the topic.
 - **[Pairings or Bilinear Maps by Alin Tomescu](https://alinush.github.io/2022/12/31/pairings-or-bilinear-maps.html)** - This resource begins with an introduction to the three fundamental properties of bilinear maps. Building on this foundation, it further explores applications such as the Tripartite Diffie-Hellman protocol, BLS signatures, and Identity-Based Encryption (IBE).
+- **[Pairings for beginners by Craig Costello](https://static1.squarespace.com/static/5fdbb09f31d71c1227082339/t/5ff394720493bd28278889c6/1609798774687/PairingsForBeginners.pdf)** - This resource introduces bilinear pairings and their cryptographic applications in a beginner-friendly manner.
```

Comment:
Nice resource! But let's mark this as optional as it is a bit more in-depth than some people might want (even though it does say "for beginners"). We have had feedback previously that we need to black-box more things and simply let people explore if they choose.